docs/2.4.2.md

```diff
@@ -35,6 +35,10 @@ see the *ProxySQL* [2.4.0 release notes](<https://github.com/sysown/proxysql/rel
 
     An upgrade from ProxySQL v1.x to ProxySQL v2.x requires the user to manually remove the ProxySQL v1.x packages. After that operation, the user can install ProxySQL v2.x.
 
+## Improvements
+
+When ``pxc_scheduler_handler`` launches, a lock file is created to prevent the running of multiple instances of ``pxc_scheduler_handler``. Previously, the lock file remained in the file system and prevented the handler script from running. Now, on a startup, ``pxc_scheduler_handler`` reads the Process identifier (PID) and timestamp from the lock file and checks if PID is running on a startup. If PID is still running, the newly launched ``pxc_scheduler_handler`` exits. If the PID is not running, ``pxc_scheduler_handler`` checks whether the timeout specified in `lockFileTimeout` exceeds. If the timeout exceeds, ``pxc_scheduler_handler`` removes the lock file and performs the operations. 
```

Comment:
```suggestion
When ``pxc_scheduler_handler`` launches, a lock file is created to prevent the running of multiple instances of ``pxc_scheduler_handler``. Previously, the lock file remained in the file system and prevented the handler script from running. Now, on a startup, ``pxc_scheduler_handler`` reads the Process identifier (PID) and timestamp from the lock file and checks if PID is running on startup. If PID is still running, the newly launched ``pxc_scheduler_handler`` exits. If the PID is not running, ``pxc_scheduler_handler`` checks whether the timeout specified in `lockFileTimeout` exceeds. If the timeout exceeds, ``pxc_scheduler_handler`` removes the lock file and performs the operations. 
```

Reply:
Updated